bin/brew

```diff
@@ -1,7 +1,10 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
What advantage do you get from `[[`? Doesn't feel better to me unless it's actually needed.